webapp/portlet/src/main/webapp/vue-apps/vertical-menu/components/VerticalMenuContent.vue

```diff
@@ -0,0 +1,52 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2023 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <v-app id="verticalMenuSiteDetails">
+    <v-card class="elevation-0" :class="extraClass">
```

Comment:
To remove elevation of `v-card`, you can use either `elevation="0"` or more common `flat`

```suggestion
    <v-card
      :loading="loading"
      :class="extraClass"
      min-height="50"
      flat>
```